multisrc/src/main/java/eu/kanade/tachiyomi/multisrc/zeistmanga/ZeistMangaGenerator.kt

```diff
@@ -9,7 +9,7 @@ class ZeistMangaGenerator : ThemeSourceGenerator {
 
     override val themeClass = "ZeistManga"
 
-    override val baseVersionCode: Int = 6
+    override val baseVersionCode: Int = 7
```

Comment:
This isn't the variable you need to bump. By bumping this you will cause all extensions in the generator to be updated unnecessarily.

```suggestion
    override val baseVersionCode: Int = 6
```

Reply:
It applies to the multisrc ones, please do read the CONTRIBUTING.md guide carefully.